src/SparkTestBase.sol

```diff
@@ -147,78 +247,9 @@ contract SparkTestBase is ProtocolV3_0_1TestBase {
         );
     }
 
-    function _variableBorrowFlowAllCollaterals(
-        ReserveConfig[] memory configs,
-        IPool pool,
-        address user
-    )
-        internal
-    {
-        for (uint256 i = 0; i < configs.length; i++) {
-            ReserveConfig memory collateral = configs[i];
-
-            if (
-                !collateral.usageAsCollateralEnabled ||
-                collateral.stableBorrowRateEnabled   ||
-                collateral.isFrozen
-            ) {
-                console.log("\n\n\n");
-                console.log("--------");
-                console.log('SKIP: COLLATERAL_DISABLED_OR_STABLE %s', collateral.symbol);
-                console.log("--------");
-                continue;
-            }
-
-            console.log("\n\n\n");
-            console.log("--------");
-            console.log("COLLATERAL %s", collateral.symbol);
-            console.log("--------");
-
-            uint256 HUNDRED_MIL = 100_000_000 * 10 ** collateral.decimals;
-
-            uint256 supplyCap
-                = collateral.supplyCap == 0 ? type(uint256).max : collateral.supplyCap;
-
-            // If supply cap is 0, deposit 100M, else limit to supply cap
-            uint256 depositAmount = HUNDRED_MIL > supplyCap ? supplyCap : HUNDRED_MIL;
-
-            _deposit(collateral, pool, user, depositAmount);
-
-            for (uint256 j = 0; j < configs.length; j++) {
-                ReserveConfig memory borrow = configs[j];
-
-                if (!borrow.borrowingEnabled || borrow.isFrozen) {
-                    console.log('\nSKIP: BORROWING_DISABLED %s', borrow.symbol);
-                    continue;
-                }
-
-                console.log("\nBORROW", borrow.symbol);
-
-                uint256 amount = 10 ** borrow.decimals;
-
-                // Add some supply for user to borrow
-                _deposit(borrow, pool, EOA, amount * 2);
-
-                // Borrow one unit of borrow token
-                this._borrow(borrow, pool, user, amount, false);
-            }
-        }
-    }
-
-    function sparkE2eTest(IPool pool, address user) public {
```

Comment:
Why are we deleting the E2E test? I was gonna say we should throw it in as part of testing this spell